src/Parsers/ParserCreateIndexQuery.cpp

```diff
@@ -87,18 +87,21 @@ bool ParserCreateIndexQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Expect
         return false;
 
      if (!parseDatabaseAndTableAsAST(pos, expected, query->database, query->table))
-         return false;
+        return false;
```

Comment:
@kitaisreal, why did you remove space from here? Now there are 3 spaces, not 4 as it should be.

Reply:
In this file there is some issue with formatting. Follow up #38720.